src/_igraph/graphobject.c

```diff
@@ -12902,6 +12981,34 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
    "  multiplying by the number of vertices minus one.\n"
    "@return: the calculated closenesses in a list\n"},
 
+  /* interface to igraph_harmonic_centrality */
+  {"harmonic_centrality", (PyCFunction) igraphmodule_Graph_harmonic_centrality,
+   METH_VARARGS | METH_KEYWORDS,
+   "harmonic_centrality(vertices=None, mode=ALL, cutoff=None, weights=None,\n"
+   "          normalized=True)\n\n"
+   "Calculates the harmonic centralities of given vertices in a graph.\n\n"
+   "The harmonic centerality of a vertex measures how easily other\n"
+   "vertices can be reached from it (or the other way: how easily it\n"
+   "can be reached from the other vertices). It is defined as the\n"
+   "mean inverse distance to all other vertices.\n\n"
+   "If the graph is not connected, and there is no path between two\n"
+   "vertices, the inverse distance is taken to be zero.\n\n"
+   "@param vertices: the vertices for which the harmonic centrality must\n"
+   "  be returned. If C{None}, uses all of the vertices in the graph.\n"
+   "@param mode: must be one of L{IN}, L{OUT} and L{ALL}. L{IN} means\n"
+   "  that the length of the incoming paths, L{OUT} means that the\n"
+   "  length of the outgoing paths must be calculated. L{ALL} means\n"
+   "  that both of them must be calculated.\n"
+   "@param cutoff: if it is an integer, only paths less than or equal to this\n"
```

Comment:
There is no requirement for this to be an integer (we can have weighted graphs, and the `<= cutoff` considered rule is valid even for unweighted)